src/frontend/komponenter/ManuellJournalfør/JournalpostSkjema.tsx

```diff
@@ -100,38 +98,38 @@ export const JournalpostSkjema: React.FC = () => {
             </StyledSectionDiv>
 
             <Knapperekke>
-                <FamilieKnapp
+                <Button
                     size="small"
                     variant={'secondary'}
-                    erLesevisning={false}
                     onClick={() => navigate(`/oppgaver`)}
                     disabled={skjema.submitRessurs.status === RessursStatus.HENTER}
                     icon={erLesevisning() && <ChevronLeftIcon />}
                 >
                     {erLesevisning() ? 'Tilbake' : 'Avbryt'}
-                </FamilieKnapp>
-                <FamilieKnapp
-                    size="small"
-                    variant="primary"
-                    erLesevisning={erLesevisning()}
-                    onClick={validerOgJournalfør}
-                    spinner={skjema.submitRessurs.status === RessursStatus.HENTER}
-                    loading={skjema.submitRessurs.status === RessursStatus.HENTER}
-                    disabled={skjema.submitRessurs.status === RessursStatus.HENTER}
-                >
-                    Journalfør
-                </FamilieKnapp>
+                </Button>
+                {!erLesevisning() && (
+                    <Button
+                        size="small"
+                        variant="primary"
+                        onClick={validerOgJournalfør}
+                        loading={skjema.submitRessurs.status === RessursStatus.HENTER}
+                        disabled={skjema.submitRessurs.status === RessursStatus.HENTER}
+                    >
+                        Journalfør
+                    </Button>
+                )}
 
-                <FamilieKnapp
-                    size="small"
-                    variant="primary"
-                    onClick={lukkOppgaveOgKnyttJournalpostTilBehandling}
-                    erLesevisning={!erLesevisning() || !kanKnytteJournalpostTilBehandling()}
-                    loading={skjema.submitRessurs.status === RessursStatus.HENTER}
-                    disabled={skjema.submitRessurs.status === RessursStatus.HENTER}
-                >
-                    Ferdigstill oppgave
-                </FamilieKnapp>
+                {!erLesevisning() && kanKnytteJournalpostTilBehandling() && (
+                    <Button
+                        size="small"
+                        variant="primary"
+                        onClick={lukkOppgaveOgKnyttJournalpostTilBehandling}
+                        loading={skjema.submitRessurs.status === RessursStatus.HENTER}
+                        disabled={skjema.submitRessurs.status === RessursStatus.HENTER}
+                    >
+                        Ferdigstill oppgave
+                    </Button>
```

Comment:
Denne knappen tror jeg aldri har blitt vist før fordi jeg ikke klarer å finne ut når uttrykket her blir false.
`kanKnytteJournalpostTilBehandling` returnerer true om det er behandle SED og saksbehandler er tilordnet oppgaven ELLER for alle andre journalposter: true hvis det ikke er lesevisning.

Da får vi altså:
- SED-oppgaver: hvis saksbehandler er tilordnet oppgaven får den kun opp knappen når man er i lesevisning, men man vil aldri komme i lesevisning på sin egen oppgave? Kanskje når den er ferdigstilt?
- Ellers: uttrykket blir da egentlig `!erLesevisning() || !!erLesevisning()` som alltid vil være sant, altså vil knappen alltid skjules

Må jo være meningen at knappen skal vises hvis man IKKE er i lesevisning OG saksbehandler kan knytte journalposten til behandling?

Reply:
Enig, synes det var rart. Synes det høres mer riktig ut som som du tenker

Reply:
Vi fant ut at det nok er meningen at logikken er som den er. Den skal kun dukke opp hvis det er behandle SED oppgave hvor saksbehandler er tilordnet oppgaven og den allerede er journalført